docs/CHANGELOG.md

```diff
@@ -1,11 +1,14 @@
 Changelog
 =========
 
+2.0.3 (Unreleased)
```

Comment:
@yuriimaz FYI: please add a "Unreleased" section instead adding the changes to the last release.